### DISCUSSION TITLE
Question about preprocess_train in train_text_to_image_sdxl.py

### DISCUSSION THREAD

**Discussion by 00why00** [huggingface/diffusers#7609] [Q&A] [answered]
Why transform image_column again in precomputed_dataset in code:
https://github.com/huggingface/diffusers/blob/7e39516627c69b71f8b21a2b53689028d4733b72/examples/text_to_image/train_text_to_image_sdxl.py#L923

**Comment by tolgacangoz** [comment, marked as answer]
Actually, [the one who added it also was unsure why it was necessary](https://github.com/huggingface/diffusers/pull/6745#issuecomment-1917681028). I guess the columns need rearranging.
